src/ORM/FieldType/DBClassName.php

```diff
@@ -38,7 +38,7 @@ class DBClassName extends DBEnum
      */
     public static function clear_classname_cache()
     {
-        Deprecation::notice('4.3', 'Call DBEnum::flushCache() instead');
+        Deprecation::notice('4.3.0', 'Call DBEnum::flushCache() instead');
```

Comment:
Need phpdoc here

If I'm being pedantic, "Call" should be replaced with "Use" but it doesn't materially change the meaning so I'm fine with the message as is.

Reply:
Updated